docs/get-started/allocation/allocation-change-request.md

```diff
@@ -12,6 +12,13 @@ This will bring up the detailed Quota attributes for that project as shown below
 
 ![Request Change Resource Allocation Attributes for OpenStack Project](images/coldfront-openstack-allocation-attributes.png)
```

Comment:
is this section supposed to have Institution - Specific Code?

Reply:
For new screenshot with newly added Attributes I will create a new PR.